libs/api-sdk/src/index.test.ts

```diff
@@ -600,4 +608,32 @@ describe("Bandada API SDK", () => {
             })
         })
     })
+    describe("Check Parameter", () => {
+        describe("Should not throw an error if the parameter has the expected type", () => {
```

Comment:
nit: the inner `describe`(s) does not seem to add any valuable information here, the `it`(s) seems to be enough.

Reply:
I think it's enough for this case since we are just testing string and that's fine, but we could also add other types, that's why I added a `describe` for it. I created 2 different `describe`(s), one if the parameter has the expected type and the other if it doesn't. These are the minimum cases to cover 100%. Let me know if it makes sense.

Reply:
Yes, I'm just wondering about the wordiness of the tests. imho having just one `describe` is enough (`Check Parameters`) and then leaving the `it`(s) inside with more specific text.

Reply:
Yes, that's the way I created it at first but I think it's easier to create the 2 possible options to make it easier to understand the tests inside the describe(s). But let me know if you find it confusing to remove the two describe(s).